notifications_utils/recipients.py

```diff
@@ -276,6 +276,25 @@ def rows_with_missing_data(self):
     def rows_with_message_too_long(self):
         return self._filter_rows("message_too_long")
 
+    @property
+    def rows_with_combined_variable_content_too_long(self):
+        """
+        Checks if the length of all variable, plus the length of the template content,
+        exceeds the SMS limit of 612 characters. Counts non-GSM characters as 2.
+        """
+        rows_too_long = []
+        if self.rows_as_list:
+            for row in self.rows_as_list:
+                if row.personalisation and self.template and self.template.placeholders:
+                    variable_length = 0
+                    for variable in row.personalisation.as_dict_with_keys(self.template.placeholders).values():
+                        if variable:
+                            variable_length += sum(1 if char in SanitiseSMS.ALLOWED_CHARACTERS else 2 for char in str(variable))
+                    total_length = variable_length + (len(self.template.content) if self.template else 0)
+                    if total_length > 612:
```

Comment:
The content size might change again in the future if we restrict to even less than 4 fragments. Would it be easy to make this a variable to be configured?

Reply:
I see we have the `SMS_CHAR_COUNT_LIMIT` constant, is that something we could leverage here?